test-commons/src/main/java/gov/cms/qpp/test/enums/EnumContract.java

```diff
@@ -1,53 +1,15 @@
 package gov.cms.qpp.test.enums;
 
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.util.UUID;
-
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 public interface EnumContract {
 
-	@Test
-	default void testValueOfNullThrows() throws NoSuchMethodException, SecurityException {
-		Class<? extends Enum<?>> type = getEnumType();
-		Method valueOf = type.getMethod("valueOf", String.class);
-		valueOf.setAccessible(true);
-		Assertions.assertThrows(IllegalArgumentException.class, () -> valueOf.invoke(null, null));
-	}
-
-	@Test
-	default void testValueOfInvalidThrows() throws NoSuchMethodException, SecurityException {
-		Class<? extends Enum<?>> type = getEnumType();
-		Method valueOf = type.getMethod("valueOf", String.class);
-		valueOf.setAccessible(true);
-		Assertions.assertThrows(InvocationTargetException.class, () -> valueOf.invoke(null, UUID.randomUUID().toString()));
-	}
-
-	@Test
-	default void testValueOfValid() throws Exception {
-		Class<? extends Enum<?>> type = getEnumType();
-		Enum<?> value = type.getEnumConstants()[0];
-		Method valueOf = type.getMethod("valueOf", String.class);
-		valueOf.setAccessible(true);
-		valueOf.invoke(null, value.name());
```

Comment:
Does the new version of Jacoco now ignore JVM supplied methods?

Reply:
Yes